harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -957,7 +1200,7 @@ def __init__(self, context: pytorch.PyTorchTrialContext) -> None:
 
     @abstractmethod
     def train_batch(
-        self, batch: pytorch.TorchData, epoch_idx: int, batch_idx: int
+            self, batch: pytorch.TorchData, epoch_idx: int, batch_idx: int
```

Comment:
what's going on with your formatting?  There's a bunch of extraneous-looking indents

Reply:
fixed.